apollo-client/pom.xml

```
@@ -93,5 +93,12 @@
 			<scope>test</scope>
 		</dependency>
 		<!-- end of test -->
+		<!-- spring cloud -->
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-context</artifactId>
+			<scope>provided</scope>
+		</dependency>
```

I think `optional` is more appropriate and it's better to move it up with the spring-boot dependencies.
```suggestion
		<dependency>
			<groupId>org.springframework.cloud</groupId>
			<artifactId>spring-cloud-context</artifactId>
			<optional>true</optional>
		</dependency>
```
```suggestion
		<dependency>
			<groupId>org.springframework.cloud</groupId>
			<artifactId>spring-cloud-context</artifactId>
			<scope>provided</scope>
		</dependency>
```